Formula/llvm.rb

```diff
@@ -598,6 +624,9 @@ def caveats
         return 0;
       }
     EOS
+    # Check where `scan-build` failure is coming from.
+    system bin/"clang++", "-v", "scanbuildtest.cpp"
+    system ENV.cxx, "-v", "scanbuildtest.cpp"
     assert_includes shell_output("#{bin}/scan-build clang++ scanbuildtest.cpp 2>&1"),
       "warning: Use of memory after it is freed"
```

Comment:
Ok, I have no idea why this is still failing, since running the compiler directly seems to work.

`scan-build` error here: https://github.com/Homebrew/homebrew-core/pull/106925#issuecomment-1231242308